app/views/posts/index.html.erb

```diff
@@ -1 +1,47 @@
-<h1> show the list of posts </h1>
\ No newline at end of file
+<%= stylesheet_link_tag 'style' %>
+
+<section class="user-section">
+  <div class="user-details">
+    <div class="user-img">
+      <img src="<%= @user.photo %>">
+    </div>
+    <div class='about-user'>
+      <p class="user-name"><%= @user.name %></p>
+      <p class="number-of-posts">Number of Posts: <%= @user.posts.count %></p>
+    </div>
+  </div>
+
+  <div class="user-posts">
+    <% @posts.each_with_index do |post, index| %>
+      <div class="post-details">
+        <h2>Post #<%= index + 1 %></h2>
+        <div class="post-content">
+          <a href="<%= post_path(post) %>">
+            <p><%= truncate(post.text, length: 50, separator: ' ') %></p>
+          </a>
+          <div class="c-l">
+            <p>Comments: <%= post.comments.count %>, Likes: <%= post.likes.count %></p>
+          </div>
+        </div>
```

Comment:
- [ ] Please use the most recent 5 comments method here on this page to show each of the rendered posts' 5 recent comments as per the requirements. 🙏